components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -143,8 +178,28 @@ - (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
   XCTAssertTrue(self.manager.internalManager.overlayView.accessibilityViewIsModal);
 }
 
-- (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
+- (void)testSnackbarAccessibiltyViewIsModalYesWithActionsAndWithoutVoiceOver {
+  // Given
+  MDCSnackbarMessageAction *action = [[MDCSnackbarMessageAction alloc] init];
+  action.title = @"Tap Me";
+  self.message.action = action;
+  self.manager.shouldEnableAccessibilityViewIsModal = YES;
+
+  // When
+  [self.manager showMessage:self.message];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
+  dispatch_async(dispatch_get_main_queue(), ^{
+    [expectation fulfill];
+  });
+  [self waitForExpectationsWithTimeout:3 handler:nil];
+
+  // Then
+  XCTAssertFalse(self.manager.internalManager.overlayView.accessibilityViewIsModal);
```

Comment:
If the name is "ViewIsModal**Yes**With..", shouldn't this be an XCTAssertTrue?

Reply:
Followed up offline. Addressed in a5ddbbd01590490c2cebea9e302ad8fbdd774c28.